tests/test_cloudpy.py

```diff
@@ -1,18 +1,26 @@
 # -*- coding: utf-8 -*-
+import datetime
 import json
 import os
 import unittest
 
 from xoxzo.cloudpy import XoxzoClient
 
 
-class TestXoxzoClient(unittest.TestCase):
+class TestXoxzoClientTestCase(unittest.TestCase):
+    def getenv_with_none_check(self, env):
+        val = os.environ.get(env)
+        if val == None:
+            raise Exception("Environment variable %s must be set" % env)
+        return val
+
```

Comment:
I see this as redundant. Accessing non existing item in dictionary will raise `KeyError`, which basically the same thing.